src/assets/locales/resources_en_US.json

```diff
@@ -1,2 +1,18 @@
 {
+  "expand_modal_default_header": {
+    "message": "Full view",
+    "_description": "The title of the modal for viewing a full text expansion"
+  },
+  "expand_button_text": {
+    "message": "See more",
+    "_description": "The button text of the expand button"
```

Comment:
Is this item and the next item for when text that is longer than 18 characters is truncated?

Reply:
Oops this is leftover. I'll remove it.